From 6b32ebc380fb75f1b119adbc5b890ec5690b7862 Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Wed, 12 Dec 2018 13:07:44 -0500
Subject: [PATCH] TOOLS-2117 sdcnode updates for November 2018 node.js security
 releases (busted upstream build fix) Reviewed by: Trent Mick
 <trentm@gmail.com> Approved by: Trent Mick <trentm@gmail.com>

---
 nodeconfigs.json | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/nodeconfigs.json b/nodeconfigs.json
index 1e4d04c..d1cccc5 100644
--- a/nodeconfigs.json
+++ b/nodeconfigs.json
@@ -177,7 +177,7 @@
     {
         "// image": "minimal-multiarch 18.1.0",
         "image": "1ad363ec-3b83-11e8-8521-2f68a4a34d5d",
-        "version": "v6.15.0",
+        "version": "v6.15.1",
         "patch_files": [
             "patches/tls-reload-certs-v6.x.patch"
         ],
@@ -191,7 +191,7 @@
         "// image": "minimal-multiarch 18.1.0",
         "image": "1ad363ec-3b83-11e8-8521-2f68a4a34d5d",
         "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
-        "version": "v6.15.0",
+        "version": "v6.15.1",
         "patch_files": [
             "patches/tls-reload-certs-v6.x.patch"
         ],
@@ -202,7 +202,7 @@
         "// image": "minimal-multiarch 18.1.0",
         "image": "1ad363ec-3b83-11e8-8521-2f68a4a34d5d",
         "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
-        "version": "v6.15.0",
+        "version": "v6.15.1",
         "patch_files": [
             "patches/tls-reload-certs-v6.x.patch"
         ],
@@ -282,7 +282,7 @@
     {
         "// image": "sdc-minimal-multiarch-lts 15.4.1",
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
-        "version": "v6.15.0",
+        "version": "v6.15.1",
         "patch_files": [
             "patches/tls-reload-certs-v6.x.patch"
         ],
@@ -296,7 +296,7 @@
         "// image": "sdc-minimal-multiarch-lts 15.4.1",
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
         "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
-        "version": "v6.15.0",
+        "version": "v6.15.1",
         "patch_files": [
             "patches/tls-reload-certs-v6.x.patch"
         ],
@@ -307,7 +307,7 @@
         "// image": "sdc-minimal-multiarch-lts 15.4.1",
         "image": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
         "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
-        "version": "v6.15.0",
+        "version": "v6.15.1",
         "patch_files": [
             "patches/tls-reload-certs-v6.x.patch"
         ],
-- 
2.21.0

